src/components/split_button/_split_button_control.scss

```diff
@@ -0,0 +1,68 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+// sass-lint:disable no-vendor-prefixes
+
+$splitButtonSeparatorColor: shade($ouiColorPrimary, 50%);
+
+.ouiSplitButton {
+
+  // Remove individual button style
+  .ouiButton,
+  .ouiButtonIcon {
+    border-width: 0px;
+    border-top-right-radius: 0px;
+    border-top-left-radius: 0px;
+    border-bottom-right-radius: 0px;
+    border-bottom-left-radius: 0px;
+
+    &:hover,
+    &:active,
+    &:focus {
+      -webkit-transform: none;
+      transform: none;
+    }
+  }
+
+  .ouiSplitButtonControl {
+
+    border: 1px solid $ouiColorPrimary;
```

Comment:
This color should be based off of the color of button that is selected. Otherwise other colors will have a primary border